CHANGES.md

```diff
@@ -80,6 +80,7 @@
 ## Bugfixes
 
 * Fixed locking issue when shutting down inactive bundle processors. Symptoms of this issue include slowness or stuckness in long-running jobs (Python) ([#30679](https://github.com/apache/beam/pull/30679)).
+* Fixed kafka polling issue due to short consumer polling timeout time. Changes to make the consumer polling timeout configurable for KafkaIO.Read with new command: KafkaIO.read().withConsumerPollingTimeout(Duration duration). Default timeout has been increased from 1 second to 2 seconds( [#30870](https://github.com/apache/beam/pull/30877)).
```

Comment:
Can we move this to Breaking Changes? We can reword these like: Default consumer polling timeout for `KafkaIO.Read` was increased from 1 second to 2 seconds. Use `KafkaIO.read().withConsumerPollingTimeout(Duration duration)` to configure this timeout value when necessary.